dockerfile/security/missing-user-entrypoint.dockerfile

```diff
@@ -9,6 +9,6 @@ RUN pip3 install semgrep
 # ruleid: missing-user-entrypoint
 ENTRYPOINT semgrep -f p/xss
 
-# TODO: metavar bug
-# ok: missing-user-entrypoint
+# TODO: metavar ellipses bug, this should be a finding but is a false negative
+# ruleid: missing-user-entrypoint
```

Comment:
For failing testcases that are related to the engine instead of the rule, we also have `todook` and `todoruleid`. This allows our tests to pass for now, and the rule in its current state to be published. But also allows our developers to verify what the intended behaviour is. If they make an engine update that fixes this problem, they will update the test syntax.
```suggestion
# TODO: metavar ellipses bug, this should be a finding but is a false negative
# todoruleid: missing-user-entrypoint
```